bin/start.js

```diff
@@ -97,11 +110,15 @@ if (argv.version) {
   } catch (e) {
     process.exit(1)
   }
-}
-
-var options = rc('hoodie', {}, _.mapKeys(_.omit(argv, ['argv']), function (value, key) {
-  return _.camelCase(key)
-}))
+})
+.alias('v', 'version')
+.env('hoodie')
+.epilogue('Options can also be specified as environment variables (prefixed with "hoodie_") or inside a ".hoodierc" file (json or ini).')
+.wrap(Math.min(150, yargs.terminalWidth()))
+.argv
+
+// merge with rc config
+_.defaultsDeep(options, _.omit(rc('hoodie', {}, null), ['config', 'configs']))
```

Comment:
can you explain the `_.omit` call here? Where would `.config` or `.configs` come from?